sql/core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -17,4 +17,17 @@
 
 package org.apache.spark.sql.test
 
-trait SharedSQLContext extends SQLTestUtils with SharedSparkSession
+trait SharedSQLContext extends SQLTestUtils with SharedSparkSession {
+
+  override protected val doThreadAuditInSparkFunSuite = false
+
+  protected override def beforeAll(): Unit = {
+    doThreadPreAudit()
```

Comment:
This looks like the same situation, but because this is a trait, it kinda relies on the suites to call `beforeAll` and `afterAll` correctly... if you don't want to audit all suites, you could write a comment explaining the situation.

Reply:
It's kind of similar but not the same. Comment added.